options/atlas_validate.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto2";
```

Comment:
why proto2? can we use proto3?

Reply:
I would recommend to change it to proto3 then as this is what we tend to use for toolkit and applications